File: SampleAppObjC/sources/advanced/HistorySampleVC.m
//
//  HistorySampleVC.m
//  SampleAppObjC
//
//  Copyright © 2018 BEAR SAS. All rights reserved.
//

#import "HistorySampleVC.h"
#import "CustomSDKViewController.h"

@interface HistorySampleVC ()

@end

@implementation HistorySampleVC

-(void)viewWillAppear:(BOOL)animated {
    [super viewWillAppear:animated];
    if (self.navigationController != nil) {
        [self.navigationController setNavigationBarHidden:NO animated:animated];
    }
}

-(IBAction)goBack {
    [self dismissViewControllerAnimated:YES completion:nil];
}

- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    if ([segue.identifier isEqual: @"history"]) {
        CustomSDKViewController* vc = (CustomSDKViewController*)[segue destinationViewController];
        vc.enableHistory = YES;
    }
}

@end

File: SampleApps/SampleApp/advanced/HistorySampleVC.swift
//
//  HistorySampleVC.swift
//  SampleApp
//
//  Copyright © 2018 BEAR SAS. All rights reserved.
//

import UIKit

final class HistorySampleVC: UIViewController {

    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        navigationController?.setNavigationBarHidden(false, animated: animated)
    }
    
    @IBAction func goBack() {
        dismiss(animated: true, completion: nil)
    }
    
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        if segue.identifier == "history" {
            let vc = segue.destination as! CustomSDKViewController
            vc.enableHistory = true
        }
    }
}
